pkg/operator/starter.go

```diff
@@ -190,7 +190,6 @@ func RunOperator(ctx context.Context, controllerConfig *controllercmd.Controller
 		configClient.ConfigV1().ClusterOperators(),
 		versionRecorder,
 		kubeInformersForNamespaces,
-		kubeInformersForNamespaces.InformersFor(metav1.NamespaceSystem).Core().V1().ConfigMaps().Informer(),
```

Comment:
I can't remove `metav1.NamespaceSystem` from `v1helpers.NewKubeInformersForNamespaces` because it's being used in https://github.com/openshift/cluster-openshift-apiserver-operator/blob/master/vendor/github.com/openshift/library-go/pkg/operator/apiserver/controllerset/apiservercontrollerset.go#L235

this should be changed in `master` branch since `Nodes` is a non-namespaced resources.